bin/rails

```diff
@@ -1,8 +1,11 @@
 #!/usr/bin/env ruby
 begin
-  load File.expand_path("../spring", __FILE__)
-rescue LoadError
+  load File.expand_path('../spring', __FILE__)
+rescue LoadError => e
+  raise unless e.message.include?('spring')
 end
+# This command will automatically be run when you run "rails" with Rails 3 gems installed from the root of your application.
+
 APP_PATH = File.expand_path('../../config/application',  __FILE__)
-require_relative '../config/boot'
+require File.expand_path('../../config/boot',  __FILE__)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.